tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -41,3 +45,35 @@ def users_to_deactivate(days):
         last_login__gte=datetime.now(tz=timezone.utc) - timedelta(days=days+1),
         account_approval_status=AccountApprovalStatusChoices.APPROVED,
         )
+
+def get_ofa_admin_user_emails():
+    """Return a list of OFA System Admin users."""
+    return User.objects.filter(
+        groups=Group.objects.get(name='OFA System Admin')
```

Comment:
@ttran-hub we will need to make sure that our group is set to `OFA System Admin` always to receive this email regarding the total number of access requests every day. 

@elipe17 we sometimes switch between OFA Admin and OFA System Admin, so if its possible to easily add this group, great. but we're okay if not.

Reply:
Thats an easy add. I will make that change real quick.